Description: fix FTBFS
 * Fix FTBFS:
     - do no include non-existed documentation file.
     - do not use deprecated and now not-needed g_type_init.
Author: Dmitrijs Ledkovs <dmitrij.ledkov@ubuntu.com>

--- libindicate-12.10.1.orig/Makefile.am
+++ libindicate-12.10.1/Makefile.am
@@ -26,7 +26,7 @@ EXTRA_DIST = \
 	xmldocs.make \
 	autogen.sh
 
-DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc --disable-scrollkeeper --disable-introspection
+DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc --disable-scrollkeeper
 
 DISTCLEANFILES = \
 	libindicate-*.tar.gz \
--- libindicate-12.10.1.orig/examples/indicate-and-crash.c
+++ libindicate-12.10.1/examples/indicate-and-crash.c
@@ -28,8 +28,6 @@ gboolean crashfunc (gpointer data) { *(i
 int
 main (int argc, char ** argv)
 {
-	g_type_init();
-
 	IndicateIndicator * indicator = indicate_indicator_new();
 	indicate_indicator_show(indicator);
 
--- libindicate-12.10.1.orig/examples/listen-and-print.c
+++ libindicate-12.10.1/examples/listen-and-print.c
@@ -128,8 +128,6 @@ server_removed (IndicateListener * liste
 int
 main (int argc, char ** argv)
 {
-	g_type_init();
-
 	IndicateListener * listener = indicate_listener_ref_default();
 
 	g_signal_connect(listener, INDICATE_LISTENER_SIGNAL_INDICATOR_ADDED, G_CALLBACK(indicator_added), NULL);
--- libindicate-12.10.1.orig/examples/im-client.c
+++ libindicate-12.10.1/examples/im-client.c
@@ -87,8 +87,6 @@ menuitem_click (DbusmenuMenuitem * menui
 int
 main (int argc, char ** argv)
 {
-	g_type_init();
-
 	IndicateServer * server = indicate_server_ref_default();
 	indicate_server_set_type(server, "message.im");
 	indicate_server_set_desktop_file(server, "/usr/share/applications/empathy.desktop");
--- libindicate-12.10.1.orig/examples/indicate-alot.c
+++ libindicate-12.10.1/examples/indicate-alot.c
@@ -29,8 +29,6 @@ with this program.  If not, see <http://
 int
 main (int argc, char ** argv)
 {
-	g_type_init();
-
 	IndicateIndicator * indicators[ALOT];
 	int i;
 
--- libindicate-12.10.1.orig/examples/show-hide-server.c
+++ libindicate-12.10.1/examples/show-hide-server.c
@@ -29,8 +29,6 @@ timeout_cb (gpointer data)
 int
 main (int argc, char ** argv)
 {
-	g_type_init();
-
 	IndicateServer * server = indicate_server_ref_default();
 	indicate_server_set_type(server, "message.im");
 	indicate_server_set_desktop_file(server, "/usr/share/applications/empathy.desktop");
--- libindicate-12.10.1.orig/docs/reference/libindicate-docs.sgml
+++ libindicate-12.10.1/docs/reference/libindicate-docs.sgml
@@ -13,11 +13,6 @@
 	<xi:include href="xml/server.xml" />
 </part>
 
-<part id="subclass">
-	<title>Subclasses for specific indicators</title>
-	<xi:include href="xml/indicator-message.xml" />
-</part>
-
 <part id="listeners">
 	<title>Classes for Listeners</title>
 	<xi:include href="xml/listener.xml" />
--- libindicate-12.10.1.orig/docs/reference/tmpl/libindicate-unused.sgml
+++ libindicate-12.10.1/docs/reference/tmpl/libindicate-unused.sgml
@@ -35,10 +35,26 @@ IndicateIndicatorMessage
 
 @parent_class: 
 
+<!-- ##### SIGNAL IndicateServer::indicator-added ##### -->
+<para>
+
+</para>
+
+@indicateserver: the object which received the signal.
+@arg1: 
+
 <!-- ##### SIGNAL IndicateServer::indicator-not-shown ##### -->
 <para>
 
 </para>
+
+@indicateserver: the object which received the signal.
+@arg1: 
+
+<!-- ##### SIGNAL IndicateServer::indicator-removed ##### -->
+<para>
+
+</para>
 
 @indicateserver: the object which received the signal.
 @arg1: 
